config/default.yaml

```diff
@@ -45,7 +45,7 @@ analysis:
 
 irods:
     home_path: /iplant/home
-    trash_path: /iplant/trash/home/de-irods
+    trash_path: /iplant/trash/home
```

Comment:
I also updated the default config to match the new value from the data-info service.